### PR TITLE
feat: create account method

### LINKED ISSUE #26: Create Account method
As there is a method to import a private key to Metamask in that case should be create account 
image

### PR DESCRIPTION
**Pull request by Lykhoyda**
**Short description of work done**

### PR Checklist

- [ ] I have run linter locally
- [ ] I have run unit and integration tests locally
  - [ ] Update configuration the newest version (readme and const)
- [ ] Rebased to master branch / merged master

### Changes

- _EXAMPLE - Removed basic auth from the controller_

### Issues

Closes #26

### CODE REVIEW

**Comment by mpetrunic** [comment]
Woops, accidentally marked as ready

File: src/page.ts [hunk 1]
```diff
@@ -85,6 +85,12 @@ export interface DappeteerPage<P = unknown> {
     name: string,
     callback: Function | { default: Function }
   ): Promise<void>;
+
+  type(
+    selector: string,
+    text: string,
+    options?: { delay: number }
+  ): Promise<void>;
 }
```
Comment (BeroBurny) [inline src/page.ts:94]: There is already `type` inside `DappeteerElementHandle`
https://github.com/ChainSafe/dappeteer/blob/d7c51f948d01230c04603e289375af03e872289d/src/element.ts#L1-L11

Comment (Lykhoyda) [reply src/page.ts:94]: I see, but honestly, I like to have a "type" method on top level as well, I think the code looks more intuitive:
```js
      const accountNameInput = await page.$(".new-account-create-form__input");
      await accountNameInput.type(accountName);
```
or 
```js
await page.type(".new-account-create-form__input", accountName);
```

Comment (BeroBurny) [reply src/page.ts:94]: open it as a question for other members to see and left a opinion
cc @irubido @mpetrunic

Comment (irubido) [reply src/page.ts:94]: 👍

**Review by irubido — APPROVED** [review]

**Pull request merged**